docs/pages/reference/terraform-provider/data-sources/provision_token.mdx

```diff
@@ -36,7 +36,7 @@ Optional:
 - `gcp` (Attributes) GCP allows the configuration of options specific to the "gcp" join method. (see [below for nested schema](#nested-schema-for-specgcp))
 - `github` (Attributes) GitHub allows the configuration of options specific to the "github" join method. (see [below for nested schema](#nested-schema-for-specgithub))
 - `gitlab` (Attributes) GitLab allows the configuration of options specific to the "gitlab" join method. (see [below for nested schema](#nested-schema-for-specgitlab))
-- `join_method` (String) JoinMethod is the joining method required in order to use this token. Supported joining methods include "token", "ec2", and "iam".
+- `join_method` (String) JoinMethod is the joining method required in order to use this token. Supported joining methods include: - azure - circleci - ec2 - gcp - github - gitlab - iam - kubernetes - spacelift - token - tpm
```

Comment:
This renders awkwardly. Personally, I'd stick with a comma separated list in the proto spec.